packages/docs/src/pages/en/features/icon-fonts.md

```diff
@@ -202,6 +202,51 @@ Use this tool to search for any Material Design Icons and copy them to your clip
 
 <DocIconList />
 
+#### MDI - UnoCSS
+
+You can use Vuetify's MDI icon set with [UnoCSS Preset Icon](https://unocss.dev/presets/icons) by installing the `@unocss/preset-icons` package, all your icons will be tree-shaken and only the icons you use will be included in your final CSS bundle.
+
+You need to install `unocss` and `@iconify-json/mdi` dev dependencies first:
+
+::: tabs
+
+```bash [pnpm]
+pnpm add unocss @iconify-json/mdi -D
+```
+
+```bash [yarn]
+yarn add unocss @iconify-json/mdi -D
+```
+
+```bash [npm]
+npm install unocss @iconify-json/mdi -D
+```
+
+```bash [bun]
+bun add unocss @iconify-json/mdi -D
+```
+
+:::
+
+then, configure UnoCSS in your project adding the preset (read the [UnoCSS integration section](https://unocss.dev/integrations/) for further details).
```

Comment:
If those further details (at minimum) are just to add `uno.config.ts`

```ts
import { presetIcons, defineConfig } from 'unocss'

export default defineConfig({
  presets: [
    presetIcons(),
  ],
})
```

We could include it here and only send user out if he/she needs to learn more.